File: core/src/org/rexcellentgames/burningknight/debug/HurtCommand.java
package org.rexcellentgames.burningknight.debug;

import org.rexcellentgames.burningknight.entity.creature.player.Player;

public class HurtCommand extends ConsoleCommand {
	{
		name = "/hurt";
		shortName = "/o";
	}

	@Override
	public void run(Console console, String[] args) {
		Player.instance.modifyHp(-1, null, true);
	}
}

File: core/src/org/rexcellentgames/burningknight/debug/HurtCommand.kt
package org.rexcellentgames.burningknight.debug

import org.rexcellentgames.burningknight.entity.creature.player.Player

class HurtCommand : ConsoleCommand("/hurt", "/h", "Ouch") {

	override fun run(console: Console, args: Array<String>) {
		Player.instance.modifyHp(-1, null, true)
	}
}